### ISSUE TITLE
Tracking intermittent test failure: UserEventsMultiple 

### ISSUE THREAD

**Issue by ggumen** [ni/VireoSDK#612]
UserEventsMultiple failed intermittently on AppVeyor:
https://ci.appveyor.com/project/vireobot/vireosdk/builds/24290277

for PR: https://github.com/ni/VireoSDK/pull/580
commit: https://github.com/ni/VireoSDK/pull/580/commits/8dfa528531f0ebb6d73b1029f236cbe5c000ca6a

Failure message:
```
	--- VTR Text
	+++ Vireo Output
	@@ -1,14 +1,14 @@
	 Generating User Events [A,B,B] (before event wait)
	 Short delay (50ms)
	 Waiting on events
	+(Generating User Events [B,A] in background)
	 Event case 1, User Event A Fired, EventType:1000 Index:0 data:<('Hello' 1)>
	 Waiting on events
	 Event case 3, User Event B Fired, EventType:1000 Index:0 data:<('Bonjour' 'Aloha' 123)>
	 Waiting on events
	 Event case 3, User Event B Fired, EventType:1000 Index:0 data:<('Bonjour' 'Aloha' 123)>
	 Waiting on events
	-(Generating User Events [B,A] in background)
	 Event case 3, User Event B Fired, EventType:1000 Index:0 data:<('Bonjour' 'Aloha' 123)>
	 Waiting on events
	 Event case 1, User Event A Fired, EventType:1000 Index:0 data:<('Goodbye' 2)>
	 Waiting on events
```

**Comment by ggumen** [comment]
Closing in favor of #361 (parent issue that tracks all of these)
